src/main/kotlin/io/spotfire/solver/solver/ConstraintViolationSummary.kt

```diff
@@ -1,16 +1,14 @@
 package io.spotfire.solver.solver
 
 import com.squareup.moshi.Json
+import kotlinx.serialization.Serializable
 import org.optaplanner.core.api.score.constraint.ConstraintMatchTotal
-
+@Serializable
 data class ConstraintViolationSummary(
-  @Json(name = "constraint_name")
   val constraintName: String,
```

Comment:
does this work if the json values are coming in as `snake_case` but the class properties are defined as `camelCase`?